sql/windows.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Windowed Data in SQL
+description: 'Rolling up data by time in SQL'
+---
+
+Because of the unbounded nature of event streams, 
```

Comment:
I think there needs to be more context about how windows work for this to be understandable by someone who hasn't used a streaming system before, but this is probably ok for now